app/controllers/pull_requests_controller.rb

```diff
@@ -15,16 +15,18 @@ def index
   def show
     github_user
     @pull_request = PullRequest.find(params[:id])
+    @likes_given = Like.where(github_user_id: github_user.id, likeable_type: "PullRequest")
+    @liked = @likes_given.where(likeable_id: @pull_request.id)[0]
     @reviewers_id = PullRequestReviewRequest.where(pull_request_id: @pull_request.id)
-    i = 0
+    position_reviewer = 0
     @reviewers = []
```

Comment:
Para obtener reviewers puedes hacer esto en vez de iterar en ese loop.
```suggestion
    @reviewers = GithubUser.where(id: @reviewers_id)
```

Otro tema relacionado, preferir usar [`.each`](https://www.tutorialspoint.com/ruby/ruby_iterators.htm) en vez de `loop` para loops

Reply:
voy a cerrar, estaba viendo commit por commit y dps cache que lo cambiaste